packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -548,6 +558,13 @@ export function usePress(props: PressHookProps): PressResult {
     return pressProps;
   }, [onPress, onPressStart, onPressEnd, onPressChange, onPressUp, isDisabled]);
 
+
+  useEffect(() => () => {
```

Comment:
Do we want to match the pattern here?

Reply:
ah yep, forgot to escape my parens when i searched for it, thanks